dev/reference/transactions/transaction-model.md

```diff
@@ -13,13 +13,13 @@ Similarly, functions such as `GET_LOCK()` and `RELEASE_LOCK()` and statements su
 
 **Note:**
 >
-> On the business side, remember to check the returned results of `commit` because even there is no error in the execution, there might be errors in the `commit` process.
+> On the application side, remember to check the returned results of `COMMIT` because even there is no error in the execution, there might be errors in the `COMMIT` process.
 
 ## Differences from MySQL
 
 ### Transaction retry
 
-By default, transactions that fail may automatically be retried by TiDB, which may lead to lost updates. This feature can be disabled by setting `tidb_retry_limit = 0`.
+TiDB does **not** automatically retry failed transactions by default, because the retry can cause update loss. You can enable this feature by setting `tidb_disable_txn_auto_retry = 0`.
```

Comment:
```suggestion
While not enabled by default, TiDB can automatically retry failed transactions when `tidb_disable_txn_auto_retry = 0`. This feature is now disabled by default because retry may lead to lost updates.
```